client/src/components/CFooter.style.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+export const Footer = styled.footer`
+  background-color: var(--white);
+  display: flex;
+  flex-direction: column;
+  height: 6rem;
+`;
+export const FooterWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+export const Line = styled.div`
+  display: block;
+  margin: 0 auto;
+  width: 100%;
+  border: 0.12rem solid var(--black);
+  border-radius: 1rem;
+  margin-top: 0.9rem;
+`;
+export const H1 = styled.h1`
+  font-weight: bold;
+  font-size: 1rem;
+  color: black
+  text-decoration-line:none;
+  margin-left:5px;
+`;
+export const H2 = styled.h2`
```

Comment:
This can also be an h3. It makes more sense semantically if they are the same kind of element! :)